optaplanner-core/src/test/java/org/optaplanner/core/api/score/buildin/bendable/BendableScoreTest.java

```diff
@@ -298,25 +297,4 @@ public void compareToHHSSS() {
                 scoreDefinitionHHSSS.createScore(1, Integer.MIN_VALUE, -20, 0, 0),
                 scoreDefinitionHHSSS.createScore(1, -20, Integer.MIN_VALUE, 0, 0));
     }
-
-    @Test
-    public void serializeAndDeserialize() {
-        PlannerTestUtils.serializeAndDeserializeWithXStream(
-                scoreDefinitionHSS.createScore(-12, 3400, -56),
-                output -> {
-                    assertThat(output.getInitScore()).isEqualTo(0);
-                    assertThat(output.getHardScore(0)).isEqualTo(-12);
-                    assertThat(output.getSoftScore(0)).isEqualTo(3400);
-                    assertThat(output.getSoftScore(1)).isEqualTo(-56);
-                });
-        PlannerTestUtils.serializeAndDeserializeWithXStream(
-                scoreDefinitionHSS.createScoreUninitialized(-7, -12, 3400, -56),
-                output -> {
-                    assertThat(output.getInitScore()).isEqualTo(-7);
-                    assertThat(output.getHardScore(0)).isEqualTo(-12);
-                    assertThat(output.getSoftScore(0)).isEqualTo(3400);
-                    assertThat(output.getSoftScore(1)).isEqualTo(-56);
-                });
-    }
```

Comment:
No replacement for the test?

Reply:
https://github.com/kiegroup/optaplanner/blob/master/optaplanner-persistence/optaplanner-persistence-xstream/src/test/java/org/optaplanner/persistence/xstream/api/score/buildin/bendable/BendableScoreXStreamConverterTest.java

Both the removed tests and tests located in optaplanner-persistence-xstream verify that individual score types can be (de)serialized via XStream correctly.